package.json

```diff
@@ -188,6 +188,7 @@
 		"config": "^3.3.6",
 		"debug": "^4.3.3",
 		"enhanced-resolve": "5.9.3",
+		"fast-average-color": "^6.4.2",
```

Comment:
I'm not certain if this package should be added here or should be added `packages/components/package.json`

Reply:
It should be added in `packages/components/package.json`
https://github.com/Automattic/wp-calypso/blob/f7d45d6dc43d703a2f39ec8efd9a44097cf6a25f/docs/dependency-management.md#L28